modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/Foldable.kt

```diff
@@ -71,6 +72,31 @@ interface Foldable<F> {
       }
     }
 
+  /**
+   * Transforms this [Foldable] into a new one which doesn't contain the given [element].
+   * Will return an identical [Foldable] if it doesn't contain [element] in the first place.
+   * Example:
+   * ```
```

Comment:
Check other snippets for ank documentation so it gets generated and is runnable :D